modules/installation-osp-kuryr-port-pools.adoc

```diff
@@ -22,4 +22,4 @@ Prior to installing a cluster, you can set the following parameters in the `clus
 * The `poolMaxPorts` parameter is the maximum number of free ports that are kept in the pool. A value of `0` disables that upper bound. This is the default setting.
 +
 If your OpenStack port quota is low, or you have a limited number of IP addresses on the pod network, consider setting this option to ensure that unneeded ports are deleted.
-* The `poolBatchPorts` parameter defines the maximum number of Neutron ports that can be created at once. The default value is `3`.
+* The `poolBatchPorts` parameter defines the maximum number of Neutron ports that can be created at once. Red Hat suggests the maximum number of Neutron ports to be created should be `3`.
```

Comment:
> Red Hat suggests the maximum number of Neutron ports to be created should be `3`.

I wouldn't restrict this to 3, I would leave it open to the user choice. We can mention however with a note a known OpenStack issue when multiple various Bulk Ports request are created at the same time, in other words when multiple Pods are created at the same time for different Namespaces.
https://bugzilla.redhat.com/show_bug.cgi?id=2024690

Reply:
Hi @MaysaMacedo 

Thank you! I will make these changes but wanted to confirm that the Bulk Ports issue is a Known Issue or a Bug Fix? From what I read in the [2024690](https://bugzilla.redhat.com/show_bug.cgi?id=2024690) bug:
> 
> Since the problem described in this bug report should be resolved in a recent advisory, it has been closed with a resolution of ERRATA. For information on the advisory (Release of components for Red Hat OpenStack Platform 16.2.2), and where to find the updated files, follow the link below. If the solution does not work for you, open a new bug report.
> https://access.redhat.com/errata/RHBA-2022:1001

I wanted to confirm, is this a bug fix for 4.11 and beyond or still a known issue?

Reply:
Thank you for the clarification! I have updated the documentaion to inform users of the Bulk Ports requests, as well as removed the restricted limit to leave the option for the user to choose. Please let me know if the changes looks good and thank you for your help again!